packages/jest-fake-timers/src/modernFakeTimers.ts

```diff
@@ -93,20 +92,48 @@ export default class FakeTimers {
     }
   }
 
-  useFakeTimers(): void {
-    if (!this._fakingTime) {
-      const toFake = Object.keys(this._fakeTimers.timers) as Array<
-        keyof FakeTimerWithContext['timers']
-      >;
-
-      this._clock = this._fakeTimers.install({
-        loopLimit: this._maxLoops,
-        now: Date.now(),
-        toFake,
+  useFakeTimers(fakeTimersConfig?: Config.FakeTimersConfig): void {
+    if (this._fakingTime) {
+      this._clock.uninstall();
+    }
```

Comment:
Alternative was mention here https://github.com/facebook/jest/issues/12362#issuecomment-1053650096. I am fine with either solution. It felt more intuitive to reinstall timers with different options, instead of requiring to switch to real timers. Calling `jest.useRealTimers()` is also some sort of boiler plate. Not sure which way is better, in the e2e tests I was writing it felt enough to reinstall timers after `jest.useFakeTimers()` is called